build.gradle.kts

```diff
@@ -42,6 +43,11 @@ subprojects {
   version = extra["VERSION_NAME"]!!
 
   plugins.withId("com.vanniktech.maven.publish") {
+    // Disable Javadoc jars. They're basically useless relics, but enabling this will also cause
+    // AGP to use an old version of Dokka which fails to run on the latest Java versions.
```

Comment:
Enabling (or leaving enabled) a Dokka-related setting in the vanniktech plugin causes _AGP_ to pull in an old Dokka version? Why?? 🧐